README.md

```diff
@@ -16,19 +16,30 @@ curl -o .env https://raw.githubusercontent.com/CleanSpeak/cleanspeak-containers/
 docker-compose up
 ```
 
-### Docker Images
+### Docker Images (4.x)
```

Comment:
I think in general this readme represents the latest. 
So if we want to call out `4.x` here, then maybe we need to leave the legacy instruction as `3.x`?

So.. my opinion would be:
- Full speed ahead, only discuss latest. Then no need to document `4.x`
- Account new and old by leaving old instructions and then adding new. 

Probably best to leave both in place for the time being since `4.x` will be a major upgrade and it may be a while before we get everyone off o `3.x`

Any opinion?

Reply:
If I step back for a moment, it's probably likely that customers are not going to be standing up new CleanSpeak 3.x instances anymore. They are either operating what they have or would be new and just go right for 4.0. Perhaps then I will just remove the 3.x instructions then.

Reply:
For any customer that is self-hosting, if they do any installs, it will be on CleanSpeak 3.x for a while is my guess. It took us years to get people off of `2.x` to `3.x`. 

This probably isn't the canonical reference for installation however, so if we just want this to be the most currently, I think that makes sense.

As long as the doc on `cleanspeak.com` reviews how to install `3.x` and `4.x` I think we are covered.